gems/smithy/spec/fixtures/service_index/model.smithy

```diff
@@ -56,7 +56,11 @@ operation DeleteResource {
 }
 
 @readonly
-operation ListResources {}
+operation ListResources {
+    output:= {
```

Comment:
Are we using new syntax or old syntax? I've been using old/explicit.

Reply:
Shouldn't really matter, these aren't even used by the test.  I generally prefer the newer/shorter syntax for readability though.

Reply:
I think we should pick one or the other for consistency.

Reply:
I prefer old syntax as it's explicit and when I search for ListResourcesOutput, I can find it in the model.